test/WebSites/PrecompilationWebSite/Startup.cs

```diff
@@ -12,7 +13,9 @@ public class Startup
         public void ConfigureServices(IServiceCollection services)
         {
             // Add MVC services to the services container
-            services.AddMvc();
+            services
```

Comment:
For now, make an extension method on `IMvcBuilder` **and** on `IServiceCollection`. The `IServiceCollection` part will go away soon.